### PR TITLE
Add continuous integration

### PR DESCRIPTION
**Pull request by jendap**
Working continuous integration!

Checkout the bazel work. I'm no bazel guru. Hopefully it is ok.

Enjoy!

BTW: The master build image will go green once you merge this ;-)

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -22,7 +22,12 @@ git_repository(
 
 bind(
     name = "protobuf_clib",
-    actual = "@tf//google/protobuf:protobuf",
+    actual = "@tf//google/protobuf:protoc_lib",
+)
+
+bind(
+    name = "protobuf_compiler",
```
Comment (kirilg) [inline WORKSPACE]: Since we're going to use grpc defined in Tensorflow's workspace, we can remove protobuf_compiler and protobuf_clib entirely from here right?

File: tensorflow_serving/servables/tensorflow/session_bundle_config.proto [hunk 2]
```diff
@@ -1,7 +1,7 @@
 syntax = "proto3";
 
+import "tensorflow/core/protobuf/config.proto";
```
Comment (kirilg) [inline tensorflow_serving/servables/tensorflow/session_bundle_config.proto]: Due to how replacements happen when exporting to open source, the order is not preserved. Please undo the changes to reorder imports in alpha order since it'll just get overwritten the next time we push anyway. (Same for a few other files that reorder imports)

**Comment by kirilg** [comment]
Thanks for this change Jan!
Please sync to get the latest changes that were just pushed. Looks good overall!

File: tensorflow_serving/servables/tensorflow/session_bundle_config.proto (hunk 2 above)
Comment (jendap) [inline tensorflow_serving/servables/tensorflow/session_bundle_config.proto]: Sure. Sorry. My bad. This is indeed wrong order for both oss and internal trees.

File: WORKSPACE (hunk 1 above)
Comment (jendap) [inline WORKSPACE]: I agree. I was happy to make it work any way.

But now I find out how to do it right :) See next commit...

**Comment by jendap** [comment]
Ok, I see, there were changes upstream...

The commits are rebased now, conflicts resolved, comments addressed.

File: tensorflow_serving/g3doc/setup.md [hunk 3]
```diff
@@ -112,3 +112,21 @@ bazel test tensorflow_serving/...
...
+
+### Continuous integration build
+
+Our [continuous integration build](http://ci.tensorflow.org/view/Serving/job/serving-master-cpu/)
+using tensorflow [ci_build](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/tools/ci_build)
+infrastructure offer you simplified development using docker. All you need is
+git and docker. No need to install all other dependencies manually.
+
+~~~shell
+git clone --recursive https://github.com/tensorflow/serving
```
Comment (kirilg) [inline tensorflow_serving/g3doc/setup.md]: git clone --recurse-submodules https://github.com/tensorflow/serving

**Comment by kirilg** [comment]
Thanks!
LGTM, just one small comment. Please squash all commits and I'll merge.

**Comment by jendap** [comment]
Sure. Done.
